modules/developer_manual/pages/general/backporting.adoc

```diff
@@ -62,17 +61,43 @@ fi
 
 commit=$1
 targetBranch=$2
-echo "backporting $commit to $targetBranch"
+
+is_merged=$(git branch --contains $1 | grep -oP '(?<=\*).*')
+
+if [ -z "$is_merged" ]; then
+    echo "$commit has not been merged. Exiting"
+    echo
+    exit
+fi
+
+pull_id=$(git log $1^! --oneline 2>/dev/null | tail -n 3 | grep -oP '(?<=#)[0-9]*')
+targetCommit=$targetBranch-$commit-$pull_id
```

Comment:
I'd wrap this in quotes, just to be sure.

Reply:
Done